src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,268 @@
-
-The following fields are retained: `cluster_name`, `container_name`, `namespace_name`, `pod_name`, `stream`, `message` and `log`.
-
-<DoNotTranslate>**Low Data Mode Log Example**</DoNotTranslate>
-
-The following log record contains all the attributes typically captured by the [New Relic Logs Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/):
-
-```json
-[
-  {
-    "cluster_name": "api-test",
-    "kubernetes": {
-      "annotations": {
-        "kubernetes.io/psp": "eks.privileged"
-      },
-      "container_hash": "fryckbos/test@sha256:5b098eaf3c7d5b3585eb10cebee63665b6208bea31ef31a3f0856c5ffdda644b",
-      "container_image": "fryckbos/test:latest",
-      "container_name": "newrelic-logging",
-      "docker_id": "134e1daf63761baa15e035b08b7aea04518a0f0e50af4215131a50c6a379a072",
-      "host": "ip-192-168-17-123.ec2.internal",
-      "labels": {
-        "app": "newrelic-logging",
-        "app.kubernetes.io/name": "newrelic-logging",
-        "controller-revision-hash": "84db95db86",
-        "pod-template-generation": "1",
-        "release": "nri-bundle"
-      },
-      "namespace_name": "nrlogs",
-      "pod_id": "54556e3e-719c-46b5-af69-020b75d69bf1",
-      "pod_name": "nri-bundle-newrelic-logging-jxnbj"
-    },
-    "message": "[2021/09/14 12:30:49] [ info] [engine] started (pid=1)\n",
-    "plugin": {
-      "source": "kubernetes",
-      "type": "fluent-bit",
-      "version": "1.8.1"
-    },
-    "stream": "stderr",
-    "time": "2021-09-14T12:30:49.138824971Z",
-    "timestamp": 1631622649138
-  }
-]
+In this example only namespaces with the label `newrelic.com/scrape` set to `true` will be scraped:
+
+```yaml
+global:
+  licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+  cluster: _K8S_CLUSTER_NAME_
+
+# ... Other settings 
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  # ... Other settings 
+  common:
+    config:
+      namespaceSelector:
+        matchLabels:
+          newrelic.com/scrape: "true"
 ```
 
-This is how the previous log record would look like after enabling `lowDataMode`:
-
-```json
-[
-  {
-    "cluster_name": "api-test",
-    "container_name": "newrelic-logging",
-    "namespace_name": "nrlogs",
-    "pod_name": "nri-bundle-newrelic-logging-jxnbj",
-    "message": "[2021/09/14 12:30:49] [ info] [engine] started (pid=1)\n",
-    "stream": "stderr",
-    "timestamp": 1631622649138
-  }
-]
+You can also use Kubernetes expressions to include or exclude namespaces using this syntax:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchExpressions:
+      - {key: newrelic.com/scrape, operator: NotIn, values: ["false"]}
+      - {key: key1, operator: In, values: ["value1"]}
 ```
 
-### New Relic Pixie Integration [#nr-pixie-integration]
+<Callout variant="tip">
+  The expressions under `matchExpressions` are concatenated.
+</Callout>
+
```

Comment:
I would add an h3 heading here like this: `### Example: Exclude namespaces [#exclude]`

Reply:
Done although the h3 heading is as follows:
`Examples with namespaces`